### PR TITLE
Implement vineyardctl put command

### LINKED ISSUE #1469: Implement the `vineyardctl put` command with the golang sdk
Describe your problem
---------------------

Although we already have the [python client APIs](https://v6d.io/notes/references/python-api.html#vineyard.RPCClient.put) for putting different types of data, there is no context during the bash execution. Besides, the current golang SDK only support to input a blob(stored as `[]byte`). As a result, the issue is to implement `vineyardctl put` command which can only put the basic data type and file as follows:

- [ ] `string`, `int`, etc.
- [ ] `slice`
- [ ] `file`

Additional context
------------------

[Golang SDK](https://github.com/v6d-io/v6d/tree/main/go/vineyard).
[Vineyardctl](https://github.com/v6d-io/v6d/tree/main/k8s/cmd).

### PR DESCRIPTION
**Pull request by zhuyi1159**
What do these changes do?
-------------------------

as titled

Related issue number
--------------------

Fixes #1469

### CODE REVIEW

**Review by sighingnow — APPROVED** [review]

File: k8s/cmd/README.md [hunk 1]
```diff
@@ -2117,3 +2162,6 @@ vineyardctl schedule workload [flags]
       --vineyardd-namespace string   the namespace of vineyard cluster (default "vineyard-system")
 ```
 
+er (default "vineyard-system")
+```
+
```
Comment (dashanji) [inline k8s/cmd/README.md:2167]: Please check the generated doc manually.

Comment (zhuyi1159) [reply k8s/cmd/README.md]: Done.

File: k8s/cmd/commands/client/put.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+		client, ch := util.NewClient()
+		if ch != nil {
+			defer close(ch)
+		}
+
+		// Convert flags.Value (type string) to type [] byte
+		value := []byte(flags.Value)
+		object_id, err := client.PutBlob(value, uint64(len(flags.Value)))
+		if err != nil {
+			log.Printf("failed to put value: %s", err)
```
Comment (dashanji) [inline k8s/cmd/commands/client/put.go:63]:
```suggestion
			log.Errorf(err, fmt.Sprintf("failed to put value: %v", value))
```

File: k8s/cmd/commands/client/put.go [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+		}
+		objectId := fmt.Sprintf("o%016x", object_id)
+		fmt.Println("object id : ", objectId)
```
Comment (dashanji) [inline k8s/cmd/commands/client/put.go:66]:
```suggestion
		log.Output("object id : ", types.ObjectIDToString(object_id))
```

**Review by dashanji — APPROVED** [review]
LGTM.

**Pull request merged**